LICENSE

```diff
@@ -225,4 +225,8 @@ The text of each license is the standard Apache 2.0 license.
    mvnw files from https://github.com/apache/maven-wrapper Apache 2.0
    svg files from skywalking-ui/src/assets/icons: https://github.com/google/material-design-icons Apache 2.0
    ZipkinQueryHandler.java reference from zipkin2.server.internal.ZipkinQueryApiV2: https://github.com/openzipkin/zipkin  Apache 2.0
-   async-profiler parser files in server-library/library-async-profiler-jfr-parser from https://github.com/async-profiler/async-profiler/tree/v3.0 Apache 2.0
+   Arguments.java in server-library/library-async-profiler-jfr-parser reference from https://github.com/async-profiler/async-profiler/tree/master Apache 2.0
+   CallStack.java in server-library/library-async-profiler-jfr-parser reference from https://github.com/async-profiler/async-profiler/tree/master Apache 2.0
+   Classifier.java in server-library/library-async-profiler-jfr-parser reference from https://github.com/async-profiler/async-profiler/tree/master Apache 2.0
+   Index.java in server-library/library-async-profiler-jfr-parser reference from https://github.com/async-profiler/async-profiler/tree/master Apache 2.0
+   Frame.java in server-library/library-async-profiler-jfr-parser reference from https://github.com/async-profiler/async-profiler/tree/master Apache 2.0
```

Comment:
I knew these classes are lack of java package names that makes them not easy to be imported, and I added another async-profiler module that solves the package name issues https://github.com/apache/skywalking/pull/12671#discussion_r1823722484 , can you take a look?

Reply:
Yes, I have seen the `ap-loader-all` package before. I use the `async-profiler-convert` package for two reasons:

1. The main reason is that `async-profiler-convert` is sufficient. The `ap-loader-all` package not only comes with a parsing class but also contains dynamic libraries of `libAsyncProfiler` on different platforms, which makes it a little bloated.

2. At the same time, I simplified some codes. For example, the parameters in `Argument.java` were modified by me, and some unnecessary parameters were deleted.